src/parameterized_tendencies/microphysics/microphysics_wrappers.jl

```diff
@@ -53,14 +53,7 @@ function q_tot_precipitation_sources(
     qₜ::FT,
     ts,
 ) where {FT <: Real}
-    return -min(
-        max(qₜ, 0) / dt,
-        -CM0.remove_precipitation(
-            cmp,
-            PP(thp, ts),
-            TD.q_vap_saturation(thp, ts),
-        ),
-    )
+    return -min(max(qₜ, 0) / dt, -CM0.remove_precipitation(cmp, PP(thp, ts)))
```

Comment:
@trontrytel I think this changes both grid-scale and subgrid-scale?

Reply:
Yes. I bet we could calibrate the parameter value for saturation based to make things stable. But this is easier